src/site/content/en/blog/fugu-status/index.md

```diff
@@ -21,42 +16,16 @@ security, privacy, trust, and other core tenets of the web.
 You can see the full list of new and potential capabilities at
 the [Fugu API Tracker](https://goo.gle/fugu-api-tracker).
 
-## Capabilities available behind a flag {: #flag }
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
 
-These APIs are only available behind a flag. They're experimental and still
-under development. They are not ready for use in production. There's a good
-chance there are bugs, that these APIs will break, or the API surface will
-change.<a name="shape-face-text"></a>
+## Capabilities available in stable {: #in-stable }
+
+The following APIs have graduated from origin trial and are available in the
+latest version of Chrome, and in many cases other Chromium based browsers.
 
-<div class="w-table-wrapper">
-  <table>
-    <thead>
-      <tr>
-        <th>Capability</th>
-        <th>Description</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>
-          <a href="/shape-detection/">Shape&nbsp;Detection (Face and Text)</a>
-        </td>
-        <td>
-          Shape detection opens up standardized
-          implementations of shape detection services and exposes them through
-          a set of JavaScript interfaces. In additon to the APIs below, the
-          Barcode Detection API is available in stable. For more information,
-          see <a href="#shape-barcode">Shape Detection (Barcode)</a>.
-          <ul>
-            <li><a href="https://www.chromestatus.com/feature/5678216012365824">Face Detection API</a></li>
-            <li><a href="https://www.chromestatus.com/feature/5644087665360896">Text Detection API</a></li>
-          </ul>
-          <em>Updated April 14, 2020</em>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-</div>
+<a style="text-decoration: none;" class="w-button w-button--primary" href="https://fugu-tracker.web.app/#:~:text=Oct%2019%2C%202021)-,shipped%20%23,-Web%20Bluetooth%20API">
+  All already shipped APIs
+</a>
```

Comment:
Should this be a button or a link?

Can you also ping Sam and see if he can fix the link so it works properly? :)

Reply:
These are links that shall look like buttons. In my tests they all worked, but the proper implementation depends on @Snugug merging https://github.com/GoogleChromeLabs/fugu-tracker/pull/26.